.github/workflows/test.yml

```diff
@@ -0,0 +1,19 @@
+name: Test
+
+on:
+  push:
+    branches-ignore:
+      - main
```

Comment:
Wenn ich das yaml richtig deute, werden damit die Tests nicht ausgeführt, wenn nach master gepusht wird. Korrekt?

Aber hatten wir nicht gesagt, dass wir bei den kleineren Libraries mit wenigen Tests die Tests immer ausführen und das nur bei der Core Library evtl. ausschalten wollen?

Reply:
Ja, aber schau nochmal genauer in die main pipeline.